src/table/components/TablePaginationActions.jsx

```diff
@@ -10,12 +13,38 @@ import { focusConfirmButton } from '../utils/handle-accessibility';
 
 const useStyles = makeStyles({
   root: {
-    display: 'flex',
+    flexShrink: 0,
+    paddingLeft: '24px',
+    transform: 'translate(0px, 4px)',
   },
   disabled: {
     color: 'rgba(0, 0, 0, 0.3)',
     cursor: 'default',
   },
+  caption: {
+    fontSize: '14px',
+    color: 'inherit',
+    width: 'fit-content',
+    position: 'relative',
+    transform: 'translate(0px, 14px)',
+    paddingRight: '8px',
+    height: '30px',
+  },
+  formControl: {
+    flexDirection: 'row',
+  },
+  hidden: {
+    display: 'none',
+  },
+  dropdown: {
+    cursor: 'pointer',
+    minWidth: '16px',
+    maxHeight: '32px',
+    transform: 'translate(0px, 6px)',
+  },
+  input: {
+    padding: '6px 32px 7px 8px',
```

Comment:
These values are a bit weird. But since there is no real UX design yet, this is fine for :)

Reply:
yeah, they're taken directly from the rows per page dropdown so that they look the same, but it is a bit messy with all the random css